syslog-ng-helm-chart/templates/compress-job.yaml

```diff
@@ -0,0 +1,56 @@
+{{- if .Values.compressor.enabled }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: job-script
+data:
+  entrypoint.sh: |-
+      #!/bin/bash
+      ls -lh /var/log/*-syslog
+      for filename in /var/log/*-syslog; do
+       if [[ $(find "$filename" -mtime +{{ .Values.compressor.retention_days }} -print) ]]; then
```

Comment:
I think there should be two stages of retention,

1) data should be compressed after a day
2) compressed files should be removed after the retention period

Reply:
I agree on this idea and i will implement it but for now this requieres another iteration on this chart.